download.dd

```diff
@@ -149,7 +149,7 @@ $(DOWNLOAD_OTHER $(GENTOO), $(LINK2 https://wiki.gentoo.org/wiki/Dlang, Gentoo),
 
 $(DOWNLOAD_OTHER $(HOMEBREW), $(LINK2 https://formulae.brew.sh/formula/dmd, Homebrew), $(CONSOLE brew install dmd))
 
-$(DOWNLOAD_OTHER $(NIX), Nix/NixOS, $(CONSOLE nix-env -i dmd))
+$(DOWNLOAD_OTHER $(NIX), $(LINK2 https://search.nixos.org/packages?query=dmd, Nix/NixOS), $(CONSOLE nix-env -iA nixpkgs.dmd))
```

Comment:
You can change the URL so that the package is shown like this:
```suggestion
$(DOWNLOAD_OTHER $(NIX), $(LINK2 https://search.nixos.org/packages?show=dmd&query=dmd, Nix/NixOS), $(CONSOLE nix-env -iA nixpkgs.dmd))
```